Editor/Modules/FogModeAnalyzer.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Unity.ProjectAuditor.Editor.Core;
+using Unity.ProjectAuditor.Editor.Diagnostic;
+using Unity.ProjectAuditor.Editor.Modules;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    class FogModeAnalyzer : ISettingsModuleAnalyzer
+    {
+        private static readonly Descriptor k_fogModeDescriptor = new Descriptor(
+            "PAS1003",
+            "Graphics: Fog Mode",
+            new[] {Area.BuildSize},
+            "Enabling Fog will result in additional shader variants, thus increasing build size.",
+            "To reduce the build size, turn off Fog at <b>Window ➔ Lighting ➔ Environment ➔ Other Settings ➔ Fog</b> to <b>Mono</b>.")
```

Comment:
In 2020.3 (up to 2023.1) versions I spot one more "Rendering" menu the user has to click.

So this looks even longer:  "Window ➔ **Rendering** ➔ Lighting ➔ Environment ..."

Reply:
Later I spotted something different in older versions:

In 2019 versions (before Unity 2020) the menus/requester look different in two places, I found "Fog" here:

"Window ➔ Rendering ➔ Lighting **Settings** ➔ **Scene** ➔ Other Settings ➔ Fog"

We could check with @mtrive if it is worth supporting. Using directives like `#if UNITY_2020_1_OR_NEWER` around the newer text it would be easy to define only this part of the Descriptor differently.

Reply:
(line removed)